packages/regenesis-surgery/test/provider.ts

```diff
@@ -73,6 +74,8 @@ export class GenesisJsonProvider implements AbstractProvider {
     if (this.state === null) {
       throw new Error('Must initialize with genesis or state object')
     }
+
+    this._isProvider = false
```

Comment:
super nit, `isLiveProvider` instead?

Reply:
This is part of the `Provider` interface